### PR TITLE
Hybrid Search

### PR DESCRIPTION
**Pull request by yuhongsun96**

### CODE REVIEW

File: backend/danswer/document_index/vespa/index.py [hunk 1]
```diff
@@ -615,7 +612,7 @@ def keyword_retrieval(
             "query": final_query,
             "input.query(decay_factor)": str(DOC_TIME_DECAY * decay_multiplier),
             "hits": num_to_retrieve,
-            "num_to_rerank": 10 * num_to_retrieve,
```
Comment (yuhongsun96) [inline backend/danswer/document_index/vespa/index.py:618]: This was never doing anything :/

File: backend/danswer/document_index/vespa/index.py [hunk 2]
```diff
@@ -640,7 +637,6 @@ def semantic_retrieval(
             # needed for highlighting while the N-gram highlighting is broken /
             # not working as desired
             + f'or ({{defaultIndex: "{CONTENT_SUMMARY}"}}userInput(@query)))'
-            + _build_vespa_limit(num_to_retrieve)
```
Comment (yuhongsun96) [inline backend/danswer/document_index/vespa/index.py:643]: As far as testing shows, adding it to yql vs parameters is the same, feels easier/cleaner to just have it in the params

File: backend/danswer/document_index/vespa/index.py [hunk 3]
```diff
@@ -686,14 +711,13 @@ def admin_retrieval(
...
         )
 
         params: dict[str, str | int] = {
             "yql": yql,
             "query": query,
             "hits": num_to_retrieve,
-            "num_to_rerank": 10 * num_to_retrieve,
+            "offset": 0,
```
Comment (yuhongsun96) [inline backend/danswer/document_index/vespa/index.py:720]: Set all these as 0 just so we remember later how to do it if we decide to introduce pagination

File: backend/danswer/configs/model_configs.py [hunk 4]
```diff
@@ -21,7 +21,9 @@
 DOC_EMBEDDING_DIM = 384
 # Model should be chosen with 512 context size, ideally don't change this
 DOC_EMBEDDING_CONTEXT_SIZE = 512
-NORMALIZE_EMBEDDINGS = (os.environ.get("SKIP_RERANKING") or "False").lower() == "true"
```
Comment (yuhongsun96) [inline backend/danswer/configs/model_configs.py:24]: Oops...

**Review by Weves — APPROVED** [review]

**Pull request merged**